### DISCUSSION TITLE
read only

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#520] [Ideas]
Is possible read only messages?
For example in some band I cant send data. But meybe is possible read biuletin or nomad page on any band.

**Comment by faragher** [comment]
Yes and no.

Yes, if you're on radio you can see any other packet sent

No, the packets are all encrypted, and a lot of traffic is actually over a
link, which requires two way communication.

You'll be able to see announces and broadcasts, as well as any "group"
messages you have an encryption key for, but you wouldn't be able to read
anyone else's messages, and since you can't announce, nobody knows where to
send your message.

While you can provide your public key to encrypt single packets to send to
you, the system doesn't know how to route it, even off the sender's machine.

So while you could see packets, you'd need to write a bunch of custom code
for both the receiver and the senders, and you'd still have encryption
problems.

**Comment by LinuxinaBit** [comment]
Reticulum is (at least in the case of NomadNet pages and bulletin boards) a lot more comparable to TCP/IP than more standard radio communications.
In those use-cases, you need to send a request first to have content sent to you.

That is even ignoring the Reticulum-specific requirement for paths created by Announces  (that @faragher explained above), which make it so that you are very unlikely to be in a situation where you can receive any data without sending _something_ first.
